### DISCUSSION TITLE
[Projects Beta] Iteration Bug - @current tag returns no iteration

### DISCUSSION THREAD

**Discussion by danielBingham** [community/community#20217] [Projects and Issues] [answered]
Today is the day one of my iterations is scheduled to end.  However, I am not able to set the time at which it ends, so it is being listed as completed.  The next iteration isn't being listed as started, so my board filtered by `iteration: @current` is simply empty.

My iterations field is named "Sprint".  Sprint 3 ends today (July 8th).  Sprint 4 starts tomorrow (July 9th).  Sprint 3 is currently listed in the "Completed" column of the iterations field.  The board filtered by `Sprint: @current` is empty.  Sprint 4 is listed in the "Active" column as planned.

I can work around this for now by setting the sprint turn over day to Saturday, but it would be nice to be able to set a Sprint turn over time.  For example, I work in EST, it'd be nice to set the Sprint end time to 5 pm on Friday EST, and the next sprint start time to 9 am on Monday EST.

**Comment by Neudrino** [comment]
There are several issues describing the same problem.

The **most up-voted** one is:
- https://github.com/github-community/community/discussions/20101

What was thought to be a **fix** was mentioned to be out for testing in:
- https://github.com/github-community/community/discussions/20101#discussioncomment-3149906

The others are:
- https://github.com/github-community/community/discussions/20217
- https://github.com/github-community/community/discussions/20032
- https://github.com/github-community/community/discussions/20108
- https://github.com/github-community/community/discussions/18834

**Comment by 2percentsilk** [comment, marked as answer]
Based on testing from the community and team, we believe we have fixed this bug. Thanks to everyone for reporting it!
